Library/Formula/paket.rb

```diff
@@ -0,0 +1,29 @@
+class Paket < Formula
+  desc "Package dependency manager for .NET"
+  homepage "https://fsprojects.github.io/Paket/"
+  version "1.14.0"
+  url "https://github.com/fsprojects/Paket/archive/#{version}.tar.gz"
```

Comment:
We don't do things this way; The Cask does, but Homebrew doesn't. Should be inlined into the `url`, i.e. `"https://github.com/fsprojects/Paket/archive/1.14.0.tar.gz"`